assignment8.txt

```diff
@@ -0,0 +1,12 @@
+Drew French
+
+1)	fprintf is used to print to a file, while printf is used to output a line of text in terminal/command prompt.
```

Comment:
right - the difference is that printf prints to the standard stream (stdout), whereas you have to specify it for fprintf.